src/tools/CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ add_executable(rados ${rados_srcs})
 target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS} radosstriper)
 install(TARGETS rados DESTINATION bin)
 
+if(WITH_LIBRADOSSTRIPER)
```

Comment:
could you avoid adding options to the command line, but instead define this in `src/include/config-h.in.cmake`

Reply:
I'm not completely clear on how cmakedefine works. If I opt for that, are all of those variables still things that can be changed from the command-line/do_cmake?

Reply:
(I will RTFM in the meantime.)

Reply:
@tchaikov The idea was that we could control striper on/off from a `%bcond` switch in the spec file.

Reply:
@chardan `git grep -w WITH_BLKIN`